templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <h1>Delete Manufacturer:</h1>
+  <p>Are you sure you want to delete the Manufacturer: {{ manufacturer }}?</p>
+  <p><i>All cars with this manufacturer will be also deleted!</i></p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, SIR!" class="btn btn-danger"/>

```

Comment:
I will recommend to change the naming on the buttons, it should work in both cases man and women